Tests/PublishTests/Tests/ProjectGeneratorTests.swift

```diff
@@ -0,0 +1,43 @@
+/**
+*  Publish
+*  Copyright (c) John Sundell 2019
+*  MIT license, see LICENSE file for details
+*/
+
+import XCTest
+@testable import PublishCLICore
+
+final class ProjectGeneratorTests: PublishTestCase {
+
+    func testMakeSiteName() {
+        XCTAssertEqual(ProjectGenerator.makeSiteName(folderName: "Name"), "Name")
+    }
+    
+    func testMakeSiteNameFromLowercasedFolderName() {
+        XCTAssertEqual(ProjectGenerator.makeSiteName(folderName: "name"), "Name")
+    }
+    
+    func testMakeSiteNameFromCamelCaseFolderName() {
+        XCTAssertEqual(ProjectGenerator.makeSiteName(folderName: "CamelCaseName"), "CamelCaseName")
+    }
+    
+    func testMakeSiteNameFromFolderNameWithNonLetters() {
+        XCTAssertEqual(ProjectGenerator.makeSiteName(folderName: "My website 1"), "Mywebsite")
```

Comment:
Is see this behaviour was in the original implementation as well, is it required?  Could the folder name be used as is - perhaps with quotes escaped?

folder name: `my "COOL" WeBsIt3` -> site_name `my &quot;COOL&quot; WeBsIt3`

Sorry if I have missed the original intent of including only letters but it seems like it could be valid to have other characters here.

> og:site_name - If your object is part of a larger web site, the name which should be displayed for the overall site. e.g., "IMDb".
https://ogp.me/

Reply:
Swift struct names can't start with digits and can't contain spaces. 
Technically the generated name can contain non-letters, but I think that this requirement (site names shouldn't contain non-letters) helps to keep things simple.

Reply:
Also keep in mind that this is just for the `publish new` convenience command. A user can easily customize the generated project to include numbers, if they want.